sql/core/src/test/scala/org/apache/spark/sql/RowSuite.scala

```diff
@@ -123,4 +123,17 @@ class RowSuite extends SparkFunSuite with SharedSparkSession {
       parameters = Map("methodName" -> "fieldIndex", "className" -> "Row", "fieldName" -> "`foo`")
     )
   }
+
+  test("SPARK-42307 - Trying to get the value from a null column should result in error") {
```

Comment:
nit:
```suggestion
  test("SPARK-42307: get the value from a null column should result in error") {
```

Reply:
Thanks for the comment @allisonwang-db. I updated the test description.

Reply:
My bad I clicked the revoke review by mistake...

Reply:
cc @allisonwang-db @cloud-fan @MaxGekk. Not sure if you got a chance to review this small PR? Thanks!